release-notes.md

```diff
@@ -2,6 +2,21 @@
 
 Kubecost Cloud does not release updates on a scheduled timeline like our self-hosted product. Instead, Kubecost Cloud product updates will be documented here periodically.
 
+## 4/25/24
+
+### New Features:
+* CloudCosts page: performance improvments through pagination and more performant api calls.
+* CloudCosts page: add auto-complete for filters and aggregation by label.
+* Clusters page: Split cluster id and cluster name on clusters page.
+* Clusters page: Enhance "Add Cluster" and "Update Cluster" modal.
+* Enhance azure cloud integrations to have faster and more resilient to failure.

```

Comment:
This is an internal improvement, we usually don't mention these, but I'm open to adding it :) 
```suggestion
* Improve Azure cost export downloads to be faster and more resilient to failures

```